### PR TITLE
Combined example

### PR DESCRIPTION
**Pull request by tpdi**
A demo example using several MUI Components (Paper, Toggle, RadioButton/Group, DropDownMenu, DatePicker, TimePicker, RaisedButton), showing setting of props and updating of state.
material-ui-scalajs-react

### CODE REVIEW

File: project/Build.scala [hunk 1]
```diff
@@ -127,13 +127,19 @@ object ScalajsReactComponents extends Build {
     .dependsOn(macros)
     .settings(
       name := "core",
+      addCompilerPlugin(compilerPlugin("org.scalamacros" % "paradise" % "2.0.1" cross CrossVersion.full)),
```
Comment (chandu0101) [inline project/Build.scala]: why we need this ?

File: project/Build.scala [hunk 2]
```diff
@@ -127,13 +127,19 @@ object ScalajsReactComponents extends Build {
...
       libraryDependencies ++= Seq(
         "com.github.japgolly.scalajs-react" %%% "core" % scalajsReactVersion,
         "com.github.japgolly.scalajs-react" %%% "extra" % scalajsReactVersion,
+        "com.github.japgolly.scalajs-react" %%% "ext-monocle" % scalajsReactVersion,
```
Comment (chandu0101) [inline project/Build.scala]: hmm, first of all why we need them here ? , core shouldn't depend on scalaz/monocle  ,even scalajs-react core/extra doesn't depend on scalaz etc

Comment (tpdi) [inline project/Build.scala]: Yes, sorry, I was trying to add Lenses to the demo. I've since removed this.

**Comment by chandu0101** [comment]
merged , congrats on first PR , thanks alot :+1:

**Comment by tpdi** [comment]
Thank you for putting together the components project.

**Pull request merged**